backend/app/models/comment.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: comments
+#
+#  id                                  :bigint           not null, primary key
+#  content(コメントの内容)             :text(65535)      not null
+#  created_at                          :datetime         not null
+#  updated_at                          :datetime         not null
+#  memo_id(コメントが参照するメモのID) :integer          not null
+#
+# Indexes
+#
+#  index_comments_on_memo_id  (memo_id)
+#
+class Comment < ApplicationRecord
+  validates :content, presence: true
+  validates :memo_id, presence: true
```

Comment:
IMO/FYI

`validates :memo_id, presence: true`
とすると、memo_idの値に該当するmemosテーブルのレコードが存在するかどうかまで検証してくれないので、
以下のようにした方がより堅牢なバリデーションと言えるかなと思いました〜！
```suggestion
  validates :memo, presence: true
```